### PR TITLE
Update logo and favicon

### PR DESCRIPTION
**Pull request by inyee786**
- Add common logo and favicon
- remove unwanted favicon and mule folder

screenshot from 2019-01-21 17-05-14
screenshot from 2019-01-21 17-06-31

Signed-off-by: inyee786 <intakhab.ali@mayadata.io>

### CODE REVIEW

File: src/ui/public/assets/favicons/browserconfig.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <msapplication>
         <tile>
             <square150x150logo src="/mstile-150x150.png"/>
-            <TileColor>#da532c</TileColor>
+            <TileColor>#ffc40d</TileColor>
```
Comment (ibreakthecloud) [inline src/ui/public/assets/favicons/browserconfig.xml]: can you explain this color change?

Comment (inyee786) [reply src/ui/public/assets/favicons/browserconfig.xml]: it changed now

File: src/ui/public/assets/favicons/safari-pinned-tab.svg [hunk 2]
```diff
@@ -2,18 +2,93 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
  "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
 <svg version="1.0" xmlns="http://www.w3.org/2000/svg"
- width="260.000000pt" height="260.000000pt" viewBox="0 0 260.000000 260.000000"
+ width="261.000000pt" height="261.000000pt" viewBox="0 0 261.000000 261.000000"
```
Comment (shiveshabhishek) [inline src/ui/public/assets/favicons/safari-pinned-tab.svg:5]: why do we need this increment in size?

Comment (inyee786) [reply src/ui/public/assets/favicons/safari-pinned-tab.svg:5]: This icon are generated online, from favicon generator, I get it from there

**Review by shiveshabhishek — APPROVED** [review]
lgtm

**Review by slalwani97 — APPROVED** [review]

**Pull request merged**